app/components/FilterOptions.tsx

```diff
@@ -37,6 +40,9 @@ const FilterOptions = ({
 
   const selectedLabel = selected ? `${selectedPrefix} ${selected.label}` : "";
 
+  const { t } = useTranslation();
+  const isAuthorDropdown = defaultLabel === t("Any author") ? true : false;
```

Comment:
Aesthetically it looks fine but `FilterOptions` is meant to be a generic component, it doesn't make sense to have anything related specifically to avatars or authors here. 

Instead you should add an `image` or `icon` prop and then pass the `<Avatar` component in from `UserFilter`

Reply:
I've made the changes as you asked. Passed Avatar component as an image prop in from `UserFilter`. And I'm styling the component inside the `FilterOptions` function because of scope limitation. is this good approach or it can be done in a better way ?

Reply:
Yes, sorry bad timing – the entire `user` object must be passed to `<Avatar` now

Reply:
made the changes as required. passing user object. didn't change anything else...